libp2p/protocols/rendezvous.nim

```diff
@@ -555,6 +560,10 @@ proc requestLocally*(rdv: RendezVous, ns: string): seq[PeerRecord] =
 proc request*(
     rdv: RendezVous, ns: string, l: int = DiscoverLimit.int, peers: seq[PeerId]
```

Comment:
it's better to use a more descriptive name for the parameter than `l`.

Reply:
Done.